lib/methods/import.js

```diff
@@ -1191,6 +1194,23 @@ module.exports = self => {
       }
 
       return doc;
+    },
+
+    getIdsAndTypes(docs) {
+      return docs.reduce((acc, doc) => {
+        acc.ids.push(doc._id);
+
+        const manager = self.apos.doc.getManager(doc.type);
+        if (self.isPage(manager)) {
+          acc.types.add('@apostrophecms/page');
+        }
+        acc.types.add(doc.type);
```

Comment:
In case of pages I add the generic `@apostrophecms/page` as well as the doc type. Because in some cases like in relationships `AposDocManager` the `moduleName` is the page type and not `@apostrophecms/page` like in `AposPageManager`.